go.mod

```diff
@@ -1,6 +1,6 @@
 module go.mongodb.org/mongo-driver
 
-go 1.13
+go 1.18
```

Comment:
Do we want to update documentation and the evergreen configs with this sync?

- Changing the evergreen go1.13-build task to go1.18-build
- Updating "Requirements" section of the README.md
- Updating CONTRIBUTING.md documentation 

Commit https://github.com/mongodb/mongo-go-driver/commit/183268c3084481ed8c5a80126e54f99a6a2cbe6e is where we made these updates for v1.

Reply:
Done, thanks!